web/src/app/pages/create-survey/create-survey.component.ts

```diff
@@ -134,6 +141,9 @@ export class CreateSurveyComponent implements OnInit {
       job.copyWith({name})
     );
   }
+
+  @ViewChild('loiSelection')
+  loiSelection?: LoiSelectionComponent;
```

Comment:
Consider removing this since it's not being used.

Reply:
Ah, this is being used in tests.